recipe/meta.yaml

```diff
@@ -12,20 +11,22 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: True # [py<37]
```

Comment:
this is not necessary as per our cbc.yaml

Reply:
I think we are follow up what is upstream, someone maybe can build it against its own env where py<37 does exist

Reply:
if this is a package update and not a rebuild, that selector should most definitely be removed.

Reply:
No, that's incorrect. Regardless of whether a recipe is an update or a rebuild, we need to specify the minimum Python version IF specified by the upstream project. This is because, as Marek suggests, anyone can use this recipe and use that to build the project in their environment.